hotel/site_sections/hotel.py

```diff
@@ -17,6 +17,17 @@ def index(self, session, department=None):
                                 .order_by(Attendee.full_name).all()
         }
 
+    def mark_hotel_eligible(self, session, id):
+        """
+        Force mark a non-staffer as eligible for hotel space.
+        This is outside the normal workflow, used for when we have a staffer that only has an attendee badge for
+        some reason, and we want to mark them as being OK to crash in a room.
+        """
+        attendee = session.attendee(id)
+        attendee.hotel_eligible = True
+        session.commit()
+        return '{} has now been overridden as being hotel eligible'.format(attendee.full_name)
```

Comment:
Something like

```python
raise HTTPRedirect('../signups/index?message={}{}', attendee.full_name, ' has now been overridden as being hotel eligible')
```

would probably be better since then the admin wouldn't need to hit the back button.  But this is a temporary fix so I'm personally cool with it for now, especially given how time sensitive this is.